docusaurus/docs/user-docs/settings/review-workflows.md

```diff
@@ -5,24 +5,54 @@ description: Managing your review workflows and stages in Strapi.
+
+<ThemedImage
+  alt="Workflow list view"
+  sources={{
+    light: '/img/assets/review-workflows/list-view-light.png',
+    dark: '/img/assets/review-workflows/list-view-dark.png',
+  }}
+/>
+
+In many organizations different teams review different parts of content. By using different review workflows for different content-types, it is possible to adjust each workflows to the needs of each team involved. The default workflow is configured to have 4 stages: To do, In progress, Ready to review, and Reviewed. All 4 stages can be edited, reordered or deleted as needed, and it is also possible to add new stages.
+
+Before being available in the [Content Manager](/user-docs/content-manager/reviewing-content), review workflows must be configured from ![Settings icon](/img/assets/icons/settings.svg) *Settings > Global settings > Review Workflows*.
 
 :::note
 - The Review workflows settings are only available to users with the Super Admin role by default. Other roles must be granted the **Review workflows** permission. See [Users, Roles, & Permissions](/user-docs/users-roles-permissions) for more information.
 - Review Workflows are only available with the Enterprise Edition. If for any reason your project was downgraded to the Community Edition, all review workflows data would not be deleted but kept until the project goes back to Enterprise.
 :::
 
+
+## Creating or editing a workflow
+
 <ThemedImage
-  alt="Review Workflows Settings"
+  alt="Workflow edit view"
   sources={{
-    light: '/img/assets/review-workflows/review-workflows.png',
-    dark: '/img/assets/review-workflows/review-workflows_DARK.png',
+    light: '/img/assets/review-workflows/edit-view-light.png',
+    dark: '/img/assets/review-workflows/edit-view-dark.png',
   }}
 />
 
-There are 4 stages in the default review workflow available in Strapi: To do, In progress, Ready to review, and Reviewed. All 4 stages can be edited, reordered or deleted as needed, and it is also possible to add new stages.
+1. Click on the **Create new workflow** button or on the edit button ![Edit icon](/img/assets/icons/edit.svg) of a workflow.
+2. In the workflow edit interface, configure the new workflow:
+
+    | Setting name   | Instructions                                                             |
+    | -------------- | ------------------------------------------------------------------------ |
+    | Name           | Write a unique name of workflow.                                         |
+    | Content-Types  | (optional) Assign this workflow to one or more existing content-types.   |
+    | Stages         | Add review stages.                                                       |
 
-## Adding a new stage
+3. Click on the **Save** button. The new workflow will be displayed in the list-view and for every content-type assigned.
+
+:::note
+- If a content-type is already assigned to another workflow, it will be re-assigned to the one currently created/ edited.
```

Comment:
Just to be sure because I read the sentence several times: does it mean that content-types can only belong to one review workflow? If so, perhaps we could rephrase a tiny bit/add another sentence afterward to make sure this point is clear for everyone. What do you think? 😊

Reply:
This has become quite a topic yesterday: https://strapihq.slack.com/archives/C04E2KAKL5T/p1689791063489499 I will try out a few things tomorrow and revisit this part. In case we are going to change that, this needs an update.

Reply:
This is now gone - the UI displays warnings and notifications for this case.